### ISSUE TITLE
Include passenger numbers

### ISSUE THREAD

**Issue by acircleda** [dshkol/airportr#8]
It would be useful if there were a function or argument that includes passenger numbers to help select the most likely airport given a city name. For example, if I choose Los Angeles and search within 50km, a number of airports are included. If that could contain (or be sorted by) passenger number, that could help in the automatic selection (via a user's function) of the most likely airport used.

**Comment by dshkol** [comment]
I agree that might be a useful thing to add to the airports data itself but would depend on the availability and quality of the pax numbers. Do you have any datasets in mind that are regularly updated and have coverage across countries?

**Comment by acircleda** [comment]
I do not. Maybe these sources can help with *some* airports:
https://en.wikipedia.org/wiki/List_of_busiest_airports_by_passenger_traffic
https://www.transtats.bts.gov/Data_Elements.aspx?Data=1 (US-Only)
